src/components/MessagesList/MessagesGroup/Message/MessagePart/PollCard.vue

```diff
@@ -39,8 +39,30 @@
 		class="poll-card"
 		role="button"
 		@click="openPoll">
+		<div v-if="showPollStatus" class="poll-card__status">
+			<NcChip
+				:variant="pollImportance"
+				no-close
+				size="small">
+				<template #icon>
+					<IconCircle
+						:size="10" />
+				</template>
+				{{ poll?.status === POLL.STATUS.OPEN ? t('spreed', 'Open poll') : t('spreed', 'Closed poll') }}
+			</NcChip>
+			<NcChip
+				v-if="poll?.resultMode === POLL.MODE.HIDDEN"
+				variant="tertiary"
+				no-close
+				size="small">
+				<template #icon>
+					<IconAccountCircleOutline :size="15" />
+				</template>
```

Comment:
I'm maybe fine with dot on a first chip, but this icon just stealing space IMO. I'd do it without:

<img width="233" height="105" alt="Image" src="https://github.com/user-attachments/assets/84c21d79-f63d-40dc-b05d-5d8ae170194c" />